src/style.css

```
@@ -1,3 +1,73 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: #fff;
+}
+
+@media (min-width: 1024px) and (max-width: 1200px) {
+  header {
+    height: 60px;
+  }
+}
+
+header {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  position: relative;
```

```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
```
no hardcode height
row in default direction
align-items center will center items vertically
horizontal paddings will make spaces
position relative is redundant here